A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_BLEED.js

```diff
@@ -79,6 +84,135 @@ var A320_Neo_LowerECAM_BLEED;
                 this.apuBleedIndication[0].setAttribute("visibility", "hidden")
                 this.apuBleedIndication[1].setAttribute("visibility", "visible")
             }
+
+            //find left pack status
+            var currentLeftPackState = SimVar.GetSimVarValue("L:A32NX_AIRCOND_PACK1_TOGGLE", "bool")
```

Comment:
Do **not** use vars, if you are getting a SimVar, or setting a variable that won't change use `const`; if you are setting a variable that will change use `let`.